Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -115,6 +118,13 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
             // TOOD: relay to the client
         }
     }
+    
+    private func setupEditor() {
```

Comment:
Small grammatical improvement. Feel free to ignore, if you prefer. 
```suggestion
    private func setUpEditor() {
```

Reply:
Updated in https://github.com/wordpress-mobile/GutenbergKit/pull/15/commits/29324052a16a172141e0821c789cc0c0595b7fcd